.github/workflows/ci-cd.yml

```diff
@@ -54,42 +54,80 @@ jobs:
         with:
           fetch-depth: 0
 
+      - if: steps.conditionals_handler.outputs.is_push_to_default_branch == 'true'
+        name: Bump GH tag
+        id: tag_generator
+        uses: mathieudutour/github-tag-action@v5.4
+        with: 
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          default_bump: false
+          release_branches: ${{ steps.data_gatherer.outputs.default_branch }}
+
```

Comment:
An important step is missing! Pay closer attention to the already existing pipelines. We will discuss this tomorrow.

Reply:
> An important step is missing! Pay closer attention to the already existing pipelines. We will discuss this tomorrow.

Resolved